src/melee/ft/types.h

```diff
@@ -492,7 +492,8 @@ typedef enum ftCommonAction {
     ASID_CAPTUREDAMAGECRAZYHAND,
     ASID_CAPTUREWAITCRAZYHAND,
     ASID_THROWNCRAZYHAND,
-    ASID_BARREL
+    ASID_BARREL,
+    ASID_MAX
```

Comment:
Maybe ASID_COUNT or similar is more accurate here? To me ASID_MAX implies the max valid ftCommonAction value, or ASID_MAX == ASID_BARREL.

Reply:
I was just following `FTKIND_MAX` but I had planned on changing them all to pascal case and `max` to `count` anyway. Just wanted the context to compile in this case.

Reply:
Ah didn't realize there was precedent here, sounds good.